content/applications/services/helpdesk.rst

```diff
@@ -32,7 +31,6 @@ team is assigned to, select it from the :guilabel:`Company` drop-down menu.
    use only.
 
    .. image:: helpdesk/team-description-webform.png
-      :align: center
       :alt: View of a Helpdesk team's website form displaying the team description.
 
 Visibility & Assignment
```

Comment:
Line 72: I would re-title this to: Follow a team's tickets

I know Odoo's field is called "Follow All Team's Tickets", but as a header, it sounds a little awkward.